packages/flutter/test/widgets/slivers_block_test.dart

```diff
@@ -315,9 +315,9 @@ void main() {
         textDirection: TextDirection.ltr,
         child: Viewport(
           offset: ViewportOffset.zero(),
-          slivers: const <Widget>[
+          slivers: <Widget>[
             SliverList(
-              delegate: SliverChildListDelegate(<Widget>[
+              delegate: SliverChildListDelegate(const <Widget>[
```

Comment:
can we use the constant constructor on these instead of the non-constant constructor?

Reply:
Since the SliverChildListDelegate non constant is the new default. Shouldn't we the test suite over?

Reply:
added 'Viewport+SliverBlock basic test with constant SliverChildListDelegate' and ''Sliver appbars const child delegate - scrolling - overscroll gap is below header''